addons/mail/static/src/core/web/thread_service_patch.js

```diff
@@ -229,20 +229,18 @@ patch(ThreadService.prototype, {
         }
         super.unpin(...arguments);
     },
-    _openChatWindow(thread, replaceNewMessageChatWindow, { openMessagingMenuOnClose } = {}) {
+    _openChatWindow(thread, replaceNewMessageChatWindow, options = {}) {
         const chatWindow = this.store.ChatWindow.insert(
             assignDefined(
                 {
+                    autofocus: true,
```

Comment:
`autofocus` needs to be an increment for the value to change, writing `true` then `true` again would not trigger the focus for example.

But with current diff you don't need to change any of this code, as you just keep the existing behavior of auto-focusing in your new flow.